main/templates/main/login.html

```diff
@@ -0,0 +1,38 @@
+{% extends "main/base_template.html" %}
+{% load crispy_forms_tags %}
+{%  load static %}
+
+{% block stylesheets %}
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/login.css' %}">
+    <title>Hotails.com - Login!</title>
+{% endblock %}
+
+{% block content %}
+
```

Comment:
You may also want to block this page from a logged-in user view (either hide the content or find a way to make it inaccessible)

Reply:
Fixed by adding "redirect_authenticated_user=True" in Login view .